src/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.ts

```diff
@@ -203,7 +209,12 @@ export default class NScaleVideoUplinkBandwidthPolicy implements VideoUplinkBand
             NScaleVideoUplinkBandwidthPolicy.targetHeightArray.length - 1
           )
         ];
-      scale = Math.max(setting.height / targetHeight, 1);
+      scale = Math.max(Math.min(setting.height, setting.width) / targetHeight, 1);
```

Comment:
[For understanding]: Is this the portrait condition handling?

Reply:
Yep